db/seed_helper.rb

```diff
@@ -1,16 +1,14 @@
 require 'open-uri'
 require 'json'
-# require 'dotenv'
-# Dotenv.load('.env')
+require 'dotenv'
 
-url = "https://api.edamam.com/api/food-database/v2/parser?app_id=#{ENV.fetch('EDAMAM_APP_ID')}&app_key=#{ENV.fetch('EDAMAM_API_KEY')}&nutrition-type=cooking"
-
-5.times do |index|
+Dotenv.load('.env') # run the script from the repo root: ruby db/seed_helper.rb
+filters= "&diet=balanced&dishType=Bread&dishType=Cereals&dishType=Main%20course&dishType=Preps&dishType=Salad&dishType=Sandwiches&dishType=Side%20dish&dishType=Soup&dishType=Starter&imageSize=THUMBNAIL"
+url = "https://api.edamam.com/api/recipes/v2?type=public&app_id=#{ENV.fetch('EDAMAM_RECIPE_API_APP_ID')}&app_key=#{ENV.fetch('EDAMAM_RECIPE_API_KEY')}#{filters}"
+4.times do |index|
```

Comment:
Why the change from `5.times` to `4.times`?

Reply:
To avoid overloading the DB since we are creating these items 8 times (for each store)- we could reduce the number of stores to increase the number of food items, but I can also revert it back to `5.times` if you prefer

Reply:
increased the num of calls in [d4b09bf](https://github.com/chrononaut76/rails-shopwise/pull/108/commits/d4b09bf61e65e35c644106ea84f92e3104de54e7)